pyproject.toml

```diff
@@ -236,7 +236,7 @@ exclude = '''
 '''
 
 [tool.pytest.ini_options]
-addopts = ["-rfEX", "-pbentoml.testing.pytest.plugin"]
```

Comment:
@aarnphm why was this in this PR anyway?

Reply:
Oh because the test in the examples will always running train everytime the test is running.

THe plugins should be for our internal test only.

Reply:
We can revert this back imo.

Reply:
Why should it be for our internal tests only?

Reply:
Given we have implemented as part of our library, it seems to me like it should be general-purpose?

Reply:
i will put up a PR to revert this.